integration-tests/pb/hello.pb.go

```diff
@@ -25,10 +25,11 @@
 package __
 
 import (
-	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
-	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 	reflect "reflect"
 	sync "sync"
+
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 )
```

Comment:
Why do you edit `*.pb.go` files? From the comment line 19:
```
// Code generated by protoc-gen-go. DO NOT EDIT.
// versions:
// 	protoc-gen-go v1.30.0
// 	protoc        v3.21.12
// source: hello.proto
```

Reply:
I runned `golangci-lint run` and it tell me failed at goimport sort, so I runned `goimport -w .` and it modified these 2 files...